### PR TITLE
docs: Integ test documentation

### PR DESCRIPTION
**Pull request by Bingjiling**
Issue #, if available:

Description of changes:
* Update documentation for integ tests
* Update openapi doc for endpoint `/api/workflows/{workflowId}/assignments` 

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: main/integration-tests/README.md [hunk 1]
```diff
@@ -1,66 +1,90 @@
 # Integration Testing for Service Workbench
 
-## Terms
+## Intro
 
-**Test Administrator:** An admin-role user added in Service Workbench specifically for creating integration test resources
+The integration-tests package is created for running API tests against all SWB APIs. 
+These tests can ran against local and dev environments during development.
```
Comment (SanketD92) [inline main/integration-tests/README.md]: typo: ...can 'run' against...

File: main/integration-tests/README.md [hunk 2]
```diff
@@ -1,66 +1,90 @@
...
 
-**Test Project:** The default project assigned for running integration tests
+## Prerequisites
 
-## Setup
+### Test Resources
 
-- Create a Test Administrator user in your Service Workbench deployment where you want to run integration tests.
-  Currently only internal auth provider can be used for authenticating this user.
+To ran integration tests, the following resources need to be created in advance:
```
Comment (SanketD92) [inline main/integration-tests/README.md]: To 'run'...

File: main/integration-tests/README.md [hunk 3]
```diff
@@ -1,66 +1,90 @@
...
-- Ensure the following GitHub secrets are created in your target repository: 
-  - AWS_ACCESS_KEY_ID
-  - AWS_SECRET_ACCESS_KEY
-  - DEPLOYMENT_BUCKET (same as `<namespace>-artifacts`)
+#### Resources to create through AWS Console:
 
+* **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(*Note the parameter name)
 
-### For Debugging Tests against Deployed Application
+* **Cost Explorer:** Turn on Cost Explorer service, activate tags tags 'Env', 'Proj' anc 'createdBy'.
```
Comment (SanketD92) [inline main/integration-tests/README.md]: 'tags' typed twice.

File: main/integration-tests/README.md [hunk 4]
```diff
@@ -1,66 +1,90 @@
...
 
 **Note**
 
-This file is unique from the rest of the `<STAGE>.yml` files created in the SDCs in that it does not gather serverless settings passed on from the hierarchies above (eg. from `main/config/settings/.defaults.yml`)
+This file is unique from other `<STAGE>.yml` files under other SDCs. It does not gather serverless settings passed on from the hierarchies above (eg. from `main/config/settings/.defaults.yml`)
+
+## Execution
 
-## Execution:
+Once test resources and config file are created, you can run the integration tests against the dev environment defined in the config file.
```
Comment (SanketD92) [inline main/integration-tests/README.md]: This is regarding the "you can run the integration tests against the **dev** environment defined in the config file" part.

The tests could be run against _any_ environment (dev, prod, local, CI etc.); what do you think of removing the 'dev' part from this string?

File: main/integration-tests/README.md [hunk 5]
```diff
@@ -1,66 +1,90 @@
...
 
-Now run the command below to trigger the integration test suite. Note that this will generate test-related resources in the deployment linked to the provided stage name.
+Note: Integration tests will create resources in the environment they executes against.
```
Comment (SanketD92) [inline main/integration-tests/README.md]: they _get executed_ against.

File: main/integration-tests/README.md [hunk 6]
```diff
@@ -1,66 +1,90 @@
...
+  - set `awsProfile` to the AWS account used for integration tests
+- Follow the steps mentioned in the `main/cicd/README.md` file to set up the CI/CD Pipeline
+- The integration test config file will be automatically saved in the deployment S3 bucket `<namespace>-artifacts` under `integration-test` folder
+- Integration test will be triggered as part of the CI/CD pipeline
+
+### Run in GitHub Workflow
+- In config file `main/integration-tests/config/settings/<STAGE>.yml`
+  - set `isBuildServer` to `true`
+  - set `isLocal` to `false`
+- Upload the config file to the deployment S3 bucket `<namespace>-artifacts` under `integration-test` folder
```
Comment (SanketD92) [inline main/integration-tests/README.md]: Might be worth mentioning that the upload step is only required if the file is already not present. (It is possible they have a CI/CD pipeline as well that pushed the config file to its S3 location)

File: main/integration-tests/README.md (hunk 4 above)
Comment (Bingjiling) [reply main/integration-tests/README.md]: I was trying to enforce the idea of not run the integ test against the prod environment. I'll update it to "against the non-production environment"?

**Review by SanketD92 — APPROVED** [review]
Good to go

**Pull request merged**